app/controllers/api/v1/orders_controller.rb

```diff
@@ -59,6 +59,26 @@ def show
           exclude_stockit_set_item: true
       end
 
+      def start_process
+        @order.start_processing(current_user) if @order.submitted?
```

Comment:
Can we move `if @order.submitted?` to state transition callback ?

```
before_transition on: :start_processing do |order|
      if @order.submitted?
        order.processed_at = Time.now
        processed_by_id = User.current_user
      end
end
```

Reply:
Thanks